Src/AmrCore/AMReX_FillPatchUtil_I.H

```diff
@@ -442,18 +489,17 @@ namespace {
                     for (MFIter mfi(mf_refined_patch); mfi.isValid(); ++mfi)
                     {
                         FAB& sfab = mf_crse_patch[mfi];
-                        FAB& dfab = mf_refined_patch[mfi];
-                        iFAB& mfab = solve_mask[mfi];
-
                         amrex::setBC(sfab.box(), amrex::convert(cgeom.Domain(), mf.ixType()),
                                      bcscomp,0,ncomp,bcs,bcr);
```

Comment:
`amrex::setBC` needs to be moved to the MFIter loop in InterpFace, because it sets BC for a single Fab.